client/components/grades-form.jsx

```diff
@@ -1 +1,83 @@
 /* eslint-disable indent */
+import React from 'react';
```

Comment:
Please re-enable the indentation checking. We're using 2-space indentation in our assignments and projects.